server/src/storage/mod.rs

```diff
@@ -189,9 +242,141 @@ impl StorageEngine {
 mod tests {
     use super::*;
     use std::path::PathBuf;
+    use std::thread;
+    use std::time::Duration;
 
+    use datafusion::arrow::array::ArrayAccessor;
```

Comment:
Unclear why `ArrayAccessor` must be imported.

Reply:
It has been removed.